sympy/physics/units/definitions/unit_definitions.py

```diff
@@ -114,6 +108,21 @@
 becquerel = Bq = Quantity("becquerel", abbrev="Bq")
 
 
+# Common mass units
+
+mg = milligram = milligrams = Quantity("milligram", abbrev="mg")
+mg.set_global_relative_scale_factor(milli, gram)
+
+ug = microgram = micrograms = Quantity("microgram", abbrev="ug", latex_repr=r"\mu\text{g}")
+ug.set_global_relative_scale_factor(micro, gram)
+
+Da = dalton = Quantity("dalton", abbrev="Da")
+Da.set_global_relative_scale_factor((1.66053906660e-27), kg)
```

Comment:
Does this work? `Da.set_globale_relative_scale_factor(amu.scale_factor/kilo, kg)`

Reply:
Yes makes, I think I should go ahead with this change .